astropy/io/misc/pandas/connect.py

```diff
@@ -42,24 +42,12 @@ def import_html_libs():
 
     global _HAS_BS4, _HAS_LXML, _HAS_HTML5LIB
 
-    try:
-        import bs4  # noqa
-        _HAS_BS4 = True
-    except ImportError:
-        pass
-
-    try:
-        import lxml  # noqa
-        _HAS_LXML = True
-    except ImportError:
-        pass
-
-    try:
-        import html5lib  # noqa
-        _HAS_HTML5LIB = True
-    except ImportError:
-        pass
-
+    from astropy.utils.compat.optional_deps import (HAS_BS4,
```

Comment:
Why not `import (HAS_BS4 as _HAS_BS4, `...

Reply:
Does that work with the `global` definitions above? If so, I agree that would be better!

Reply:
Had not seen those definitions.... I think it should still work ... and, yes, I checked quickly, it does.